build_image.sh

```diff
@@ -21,7 +24,18 @@ make clean
 make
 
 BUILD_DATE=$(date +'%Y%m%d%H%M%S')
-REPO_NAME="openebs/cstor-base"
+if [ "${ARCH}" = "x86_64" ]; then
+	REPO_NAME="openebs/cstor-base"
+	DOCKERFILE_BASE="Dockerfile.base"
+	DOCKERFILE="Dockerfile"
+elif [ "${ARCH}" = "aarch64" ]; then
+	REPO_NAME="openebs/cstor-base-arm64"
```

Comment:
we have to make sure that while building cstor-pool-mgmt image for arm64 it referes to this arm64 base image.

Reply:
Thanks @pawanpraka1 . Yeah we should use different `CSTOR_BASE_IMAGE= openebs/cstor-base:${BASE_TAG}` when building for arm64. https://github.com/openebs/maya/blob/f18cf9f32d72c1f1bddfb89ad31743ef6d720caa/GNUmakefile#L96